uaclient/cli.py

```diff
@@ -1865,6 +1865,21 @@ def _warn_about_new_version(cmd_args=None) -> None:
         logging.warning(NEW_VERSION_NOTICE.format(version=new_version))
 
 
+def _warn_about_output_redirection(cmd_args) -> None:
```

Comment:
I am wondering if we should turn this into a decorator, since right now we are running it for every cli command, but it is aimed only at two commands

Reply:
well, the command is the first check - it is a no-op for any other commands. So making the decorator is more of an implementation detail imho; we can add/remove new checks or commands in any case.
If you see some advantage in turning it into a decorator, can be done.

Reply:
I think that it makes it explicit which commands do indeed need this warning. But I agree that it bottles down to an implementation detail. So feel free to choose what you prefer